styles/global.scss

```diff
@@ -27,3 +27,10 @@ a {
 :before {
   box-sizing: border-box;
 }
+
+:target:before {
+  content: '';
+  display: block;
+  height: $header_height + 30px; /* fixed header height*/
+  margin: -($header_height + 30px) 0 0; /* negative fixed header height */
+}
```

Comment:
What this for? I'm not sure I understand.

Reply:
when you navigate from one section to another, the section is like right on the top. The title gets hidden behind the header, so that's why i put this margin.

Reply:
Oh nice!